### PR TITLE
feat: dataset view, annotation view of ArticleAnnotationProject

### PR DESCRIPTION
**Pull request by artikandri**
What's changed: 
- `.gitignore`: updated the gitignore to ignore automatically generated `assets/css/fonts.css`
- `frontend/components/tasks/metadata/ListMetadata.vue`: updated the component to show the meta content as list if its an object 
- `frontend/domain/models/example/example.ts`: added new classes for Typescript purposes
- `frontend/pages/projects/_id/dataset/index.vue`: imported ArticleList.vue
- `frontend/i18n/*/projects/dataset.js`: added translations for the newly added feature
- `frontend/services/application/example/exampleData.ts`: added new classes for Typescript purposes, added new keys to ExampleDTO to correspond with the article structure

What's new: 
- `frontend/components/example/ArticleList.vue`: New component to handle the dataset view for the articles. Allows nested view by grouping the imported data with their `article_id`. 
- `frontend/pages/projects/_id/article-annotation/index.vue`: annotation view of the ArticleAnnotationProject
- `frontend/services/application/example/exampleVuetify.ts`: new classes for Typescript purposes
- `frontend/i18n/fr/projects/comments.js`: translation of the comments file in the French language

Screenshots: 
image
image

### CODE REVIEW

**Review by haanh764 — APPROVED** [review]
I have test and everything works fine! And generally the code looks fine also. Thank you for your excellent work! Approved for merging!

**Comment by artikandri** [comment]
Sorry, I forgot that Im not supposed to work on the annotation view now, I did it because I wanted to test the annotate button and the view wasnt there so I just did it by following the custom project.  I will adjust the view later on the next PR.

**Comment by teddy-f-47** [comment]
> Sorry, I forgot that Im not supposed to work on the annotation view now, I did it because I wanted to test the annotate button and the view wasnt there so I just did it by following the custom project. I will adjust the view later on the next PR.

imo, it is totally fine and in fact it is nice XD I mean it is better that we start from the existing annotation view instead of coding from zero XD imo you don't need to change the view at all, let's keep it as it is

File: frontend/i18n/fr/projects/comments.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+export default {
+    comments: 'Commentaires',
+    removeComment: 'Supprimer le commentaire',
+    removePrompt: 'Êtes-vous sûr de vouloir supprimer ces commentaires?',
+    commentView: 'Voir',
+    created_at: 'Créé à',
+    document: 'Document',
+    send: 'Envoyer',
+    message: 'Message'
+  }
```
Comment (teddy-f-47) [inline frontend/i18n/fr/projects/comments.js]: Maybe we should add empty new line at end of file? XD

**Review by teddy-f-47 — APPROVED** [review]
I think everything looks very good, thank you for your excellent work! Also as I mentioned, I think we can keep `frontend/pages/projects/_id/article-annotation/index.vue` as it is now, and we will proceed with the 'real' annotation view from it (I think it will be easier). Also it is an excellent idea to handle the meta data as you do here. Approved for merging!

edit: I forgot that there is a minor comment but it is just styling thing XD

**Pull request merged**